static/js/ducks/analysis_services.js

```diff
@@ -8,17 +8,52 @@ const FETCH_ANALYSIS_SERVICES_OK = "skyportal/FETCH_ANALYSIS_SERVICES_OK";
 
 const REFRESH_ANALYSIS_SERVICES = "skyportal/REFRESH_ANALYSIS_SERVICES";
 
+const FETCH_ANALYSIS_SERVICE = "skyportal/FETCH_ANALYSIS_SERVICE";
```

Comment:
I would go even further and change the plurals to be something like `FETCH_ANALYSIS_SERVICES_LIST` to make it more obvious. But that's your call.